### PR TITLE
update disambiguated json output, version label based graph

### PR DESCRIPTION
**Pull request by apeters**
re #7314

### CODE REVIEW

File: arches/app/utils/label_based_graph_v2.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
+from arches.app.datatypes.datatypes import DataTypeFactory
```
Comment (chrabyrd) [inline arches/app/utils/label_based_graph_v2.py:1]: The only difference between this and V1 is the changing the `get_display_value` call to `to_json`, yeah? Should we be duplicating everything, or should we work versioning logic into the current endpoint? Or maybe follow an inheritance pattern and overwrite the one method?

**Review by chrabyrd — COMMENTED** [review]
overall looks good! Only thing I really have is that the tests from v1 should make it over to v2

**Review by chiatt — COMMENTED** [review]
I reviewed the output of each datatype and they all make sense to me. The one exception maybe, as we discussed earlier, is the 'selected' property on the domain and domain list datatypes. It seems unnecessary, but I'm not sure if it should be filtered out here or whether it ought to be removed from the config itself.

**Review by chiatt — APPROVED** [review]

**Review by chrabyrd — CHANGES_REQUESTED** [review]
just wanna make sure we don't accidentally merge before getting the v1 tests in ;-)

**Comment by apeters** [comment]
Here's a model of all datatypes to test with.

[_All Datatypes.json.txt](https://github.com/archesproject/arches/files/6864502/_All.Datatypes.json.txt)

**Comment by apeters** [comment]
@chiatt, @chrabyrd, because the timeing is critical, maybe we should write a ticket specifically for crating tests for the v2 version of label_based_graph, so we can merge this in sooner rather than later.

**Review by chrabyrd — APPROVED** [review]
@apeters agreed! It might also be a good exercise to get peoples' feet wet with testing patterns.

**Comment by chiatt** [comment]
@apeters & @chrabyrd I just noticed your latest comments. This afternoon I added a new file for the v2 tests in this PR: https://github.com/archesproject/arches/pull/7636

**Pull request merged**